### ISSUE TITLE
New driver support (version 450.66)

### ISSUE THREAD

**Issue by MastaG** [keylase/nvidia-patch#308]
**Please provide details about your environment (please complete the following information)**
- Nvidia driver version: 450.66
- GPU Model: GTX 1050Ti
- OS: doesn't matter, just need the patch for sed
- [x] I'm willing to participate in testing of new patch version

**Comment by IvanCharcos** [comment]
I am also looking for this driver support

-     Nvidia driver version: 450.66
-     GPU Model: GTX 1060
-     OS: Ubuntu 18.04
-     [ ]  I'm willing to participate in testing of new patch version

**Comment by cybrnook** [comment]
It's just two lines, but I added 450.66 here if you need it in the meantime (just for linux, since that's what I use).

https://raw.githubusercontent.com/cybrnook/nvidia-patch/master/patch.sh

**Comment by reloxx13** [comment]
@keylase small ping :p 🌭

**Comment by severd8** [comment]
Following

**Comment by SkyBehind** [comment]
> It's just two lines, but I added 450.66 here if you need it in the meantime (just for linux, since that's what I use).
> 
> https://raw.githubusercontent.com/cybrnook/nvidia-patch/master/patch.sh

Thanks for this!

**Comment by MastaG** [comment]
yeah well, I only needed to know whether the previous:
```
's/\x85\xc0\x41\x89\xc4\x75\x1f/\x31\xc0\x41\x89\xc4\x75\x1f/g'
```
was still working.

Did you verify it? e.g. can you open up more than two encoding sessions?

**Comment by Kriskras99** [comment]
I can verify that this patch works, tested up to 5 concurrent transcodes

**Comment by Kriskras99** [comment]
> It's just two lines, but I added 450.66 here if you need it in the meantime (just for linux, since that's what I use).
> 
> https://raw.githubusercontent.com/cybrnook/nvidia-patch/master/patch.sh

You should probably open a pull request, so that keylase can easily merge it.

**Comment by cybrnook** [comment]
> > It's just two lines, but I added 450.66 here if you need it in the meantime (just for linux, since that's what I use).
> > https://raw.githubusercontent.com/cybrnook/nvidia-patch/master/patch.sh
> 
> You should probably open a pull request, so that keylase can easily merge it.

Naw, because I'm only doing the Linux side. A full PR would include windows and everything else. More than I wanted to do.

**Comment by severd8** [comment]
@Snawoot Courtesy ping to see if there are any updates to support latest version. Thanks for all that you do!

**Comment by Jackfritt** [comment]
Works like a charm. Thx!

**Comment by severd8** [comment]
So no update on Windows version?

**Comment by amnesia0287** [comment]
What happened to whoever was making them and posting them before? lol. Seems like an inopportune time when new drivers for 30xx series cards are coming and gonna keep being updated for a while.

**Comment by akbaryahya** [comment]
I'm a bit confused I'm using windows 10 insider build 20215, and I'm getting driver version 460.15 on gtx 960 since that update I got error "Playback [3f046c]: AVS - SVSmoothFps: unable to init GPU-based renderer - code 196830" when using svp for gpu. I checked if there is a new patch but it seems there is not yet, and it makes me confused on official website nvidia is still using 456.38 and I check version vulkan is 452.28 and it looks like the patch version is still not available, so I thought I would try to downgrade or just wait it?

**Comment by cybrnook** [comment]
> It's just two lines, but I added 450.66 here if you need it in the meantime (just for linux, since that's what I use).
> 
> https://raw.githubusercontent.com/cybrnook/nvidia-patch/master/patch.sh

I switched to a Quadro card, so I no longer really need this. But I updated the link to support the latest Beta Linux Driver 455.23.04. For anyone who wants to try, please let me know if it works for you with more than 3 x streams.

**Comment by Snawoot** [comment]
Resolved by #312
